static/js/util/ProfileFormFields.js

```diff
@@ -1,3 +1,4 @@
+// @flow
```

Comment:
I think so, from [this](https://flowtype.org/docs/new-project.html#_) it looks like we can pass a `--all` command line flag to have it look at all JS files, and we could combine that with some `exclude` rules to avoid checking things we'd rather not bother with typechecking (for instance the dependencies that are checked-in in `static/js/public` for the homepage).